nengo/builder/signal.py

```diff
@@ -69,13 +77,15 @@ def __init__(self, initial_value,
         self._readonly = bool(readonly)
 
     def __getstate__(self):
+        # TODO: support sparse type?
```

Comment:
Shouldn't be too hard to do this. Because we don't have to worry about views, we should just be able to return the Scipy sparse matrix itself (which I assume is pickleable).